net-analyzer/goaccess/goaccess-1.1.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="A real-time web log analyzer and interactive viewer that runs in a terminal"
+HOMEPAGE="http://goaccess.io"
+SRC_URI="http://tar.goaccess.io/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="btree bzip2 debug geoip getline libressl tokyocabinet ssl unicode zlib"
+
+RDEPEND="sys-libs/ncurses:0=[unicode?]
+	geoip? ( dev-libs/geoip )
+	libressl? ( dev-libs/libressl )
+	!tokyocabinet? ( dev-libs/glib:2 )
+	tokyocabinet? (
+		dev-db/tokyocabinet[bzip2?,zlib?]
+		btree? (
+			bzip2? ( app-arch/bzip2 )
+			zlib? ( sys-libs/zlib )
+		)
+	)
+	ssl? ( dev-libs/openssl:0= )"
```

Comment:
ok so libressl is a bit trickier:
```
ssl? (
	!libressl? ( dev-libs/openssl:0= )
	libressl? ( dev-libs/libressl:0= )
)
```
USE=libressl only discriminates between openssl and libressl